nx-dev/util-ai/src/lib/utils.ts

```diff
@@ -0,0 +1,103 @@
+import OpenAI from 'openai';
+import { SupabaseClient, createClient } from '@supabase/supabase-js';
+
+export function checkEnvVariables(
```

Comment:
Since this isn't needed outside of this module, we can inline it into `getSupabaseClient` or at least not export it.